ModulBot/Cards/ActionButton.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+using System.Collections.Generic;
+using ModulBot.Interfaces;
+using ModulBot.Models;
+using ModulBot.Database.PostgresRepositories;
+using ModulBot.Commands;
+using Telegram.Bot;
+using Telegram.Bot.Types;
+using Telegram.Bot.Types.ReplyMarkups;
+using System.Threading.Tasks;
+using Npgsql;
+using ModulBot.Database;
+using Microsoft.Extensions.Configuration;
+
+namespace ModulBot.Cards
+{
+    public static class ActionButton
+    {
+        private static IConfiguration Configuration { get; }
+        public static async void DoAction(long chatId, int messageId, string[] buttonCallbackData)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 17 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS3776)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return 'Task' instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS3168)